modules/fscloud/variables.tf

```diff
@@ -79,15 +133,7 @@ variable "zone_service_ref_list" {
     ])
     error_message = "Provide a valid service reference for zone creation"
   }
-  default = ["cloud-object-storage", "codeengine", "containers-kubernetes",
-    "databases-for-cassandra", "databases-for-elasticsearch", "databases-for-enterprisedb",
-    "databases-for-etcd", "databases-for-mongodb",
-    "databases-for-mysql", "databases-for-postgresql",
-    "databases-for-redis", "directlink",
-    "iam-groups", "is", "messagehub",
-    "messages-for-rabbitmq", "schematics", "secrets-manager", "server-protect", "user-management",
-  "apprapp", "compliance", "event-notifications", "logdna", "logdnaat"]
-  description = "(List) Service reference for the zone creation"
+  description = "Service reference for the zone creation"
```

Comment:
It will be good to have little more detailed description so that user will know what is the significance of `optional(string)`.